### PR TITLE
Install script

### PR DESCRIPTION
**Pull request by dsarahstamps**
Installation script and associated document for installing development version of ASPECT on stampede

### CODE REVIEW

**Comment by tjhei** [comment]
I am confused about the location of stampede.{tex|pdf}, shouldn't they be in doc/install?

File: doc/install/aspect-setup_stampede.sh [hunk 1]
```diff
@@ -0,0 +1,101 @@
+#!/bin/bash
+# script originated from Jonathan Perry-Hauts May 2015 
+# Updated for installation on Stampede by D. Sarah Stamps 
+# at 2015 ASPECT Hackathon
```
Comment (tjhei) [inline doc/install/aspect-setup_stampede.sh]: can you add a comment that refers to stampede.pdf for more information?

**Comment by dsarahstamps** [comment]
Moved stampede.* to doc/install and updated the header of the install script to reference stampede.pdf

File: doc/install/stampede.tex [hunk 2]
```diff
@@ -0,0 +1,157 @@
...
+cd ../ \\ 
+
+Check where you are located: \\
+git branch  \\
+
+You should see:  \\
+ $*$master  \\
+ 
+ Make a local branch for your work \\
+ git branch myproject \\
```
Comment (tjhei) [inline doc/install/stampede.tex]: I think the remainder of this file should not be in here (this is basic git usage and not related to stampede)

**Comment by tjhei** [comment]
Looks good otherwise.

**Pull request merged**